deploy-edgeX.sh

```diff
@@ -25,9 +25,11 @@
 #    . $(dirname "$0")/bin/env.sh
 #fi
 
+source ./sync.sh
+
 run_service () {
 	echo "\033[0;32mStarting.. $1\033[0m"
-	docker-compose up -d $1
+        docker-compose -f $COMPOSE_FILE up -d $1
```

Comment:
Unnecessary change if we just dynamically download `docker-compose.yml` and modify it in place.

Note: be sure to add `docker-compose.yml` to .gitignore.

Reply:
The original `docker-compose.yml` is still needed, because it has the services for blackbox-testing only.  Thus, it is necessary to use `$COMPOSE_FILE` to specify the compose file from developer-script.

Reply:
I see now.  Perhaps call them `docker-compose-services-stack.yml` and `docker-compose-test-tools.yml` to distinguish the two?

Reply:
Good idea!

The original `docker-compose.yml` is changed to `docker-compose-test-tools.yml`, however, the compose file from developer-scripts still uses the name as its name in developer-scripts to make sure using the correct compose file as expected.